lsp/src/lsp-server.c

```diff
@@ -45,6 +45,7 @@
 # include "spawn/lspunixoutputstream.h"
 #endif
 
+#include  <unistd.h>
```

Comment:
Is this change on purpose?

If so, @techee should add it upstream as well.

Reply:
No problem to add but is it actually needed @giuspen ? The plugin seemed to build without it on my machine at least.

Reply:
Hi @techee have you tried to build in msys2 or you are referring to cross compiling? I couldn't on windows/mays2 without this patch but I can revert and reapply when I build locally anyway, not a big deal

Reply:
I tried it but it's possible it was before making some changes in `lsp-server.c` that require this header. I was just wondering if you ran into any problems without it.

No problem to add this include and make the corresponding change in the lsp repo.

Reply:
FWIW, it's most probably because of the `getpid()` call, whose declaration indeed is in *unistd.h*.

Reply:
The include of the `unistd.h` header has been merged via #1421.
This will lead to a merge conflict here, @giuspen maybe you want to rebase or merge the branch on current master.

Reply:
Thanks @eht16 I will rebase shortly resolving the conflict